api/datalake_api/querier.py

```diff
@@ -172,9 +174,12 @@ def _unpack(self, result):
 
 class ArchiveQuerier(object):
 
-    def __init__(self, table_name, dynamodb=None):
+    def __init__(self, table_name, latest_table_name=None, dynamodb=None):
         self.table_name = table_name
+        self.latest_table_name = latest_table_name
         self.dynamodb = dynamodb
+        self.use_latest_table = os.environ.get("DATALAKE_USE_LATEST_TABLE",
```

Comment:
`os.environ.get` will always return a string, so you probably want `self.use_latest_table = os.environ.get("DATALAKE_USE_LATEST_TABLE", "false").lower() == "true"`

Reply:
Fixed